integrations/client/test_delphi_epidata.py

```diff
@@ -178,6 +191,29 @@ def test_covidcast(self):
         'message': 'success',
       })
 
+    with self.subTest(name='request as-of a date capturing a covidcast-covidcast2 collision'):
+      # fetch data, specifying as_of
+      response_1a = Epidata.covidcast(
+          'src', 'sig', 'day', 'county', 20200414, '01234',
+          as_of=20200499)
```

Comment:
is this 04/99 intentional? (so it's in the mega future?)

Reply:
Well, you know the rhyme,
```
30 days hath September, June, and November,
All the rest have 31, 
But April's 99
especially in a pandemic year
```

Reply:
but really that's just "grab me the most recent issue in april"

Reply:
lol that would be quite the year. Perhaps leave a comment explaining?